source/admin_manual/list_of_events.rst

```diff
@@ -164,646 +164,284 @@ SQL Categories
-+==============+============================================================+
-| error        | Error string if error occurred.                            |
-+--------------+------------------------------------------------------------+
 
-Common fields
--------------
+.. dovecot_core:event:: auth_client_cache_flush_finished
+   :added: v2.3.7
+
+   :field error: Error string if error occurred.
+
+   TODO
+
+
+.. dovecot_event:field_group:: auth_client_common
+
+   :field mechanism: Name of used SASL mechanism (e.g. PLAIN).
+   :field service: Name of service. Examples: ``imap``, ``pop3``, ``lmtp``, ...
+   :field transport:
+     Transport security indicator. Values:
+       * ``insecure``
+       * ``trusted``
+       * ``TLS``
+   :field session: Session identifier.
+   :field certificate_user: Username from certificate.
+   :field client_id: Expands to client ID request as IMAP arglist. Needs
+     :dovecot_core:ref:`imap_id_retain=yes <imap_id_retain>`.
+   :field local_name: TLS SNI.
+   :field local_ip: Local IP client connected to.
+   :field remote_ip: Remote IP of client.
+   :field local_port: Local port client connected to.
+   :field remote_port: Remote port of client.
+   :field real_local_ip: Real local IP as seen by the server.
+   :field real_remote_ip: Real remote IP as seen by the server.
+   :field real_local_port: Real local port as seen by the server.
+   :field real_remote_port: Real remote port as seen by the server.
+   :field tls_cipher:  Cipher name used, e.g. ``TLS_AES_256_GCM_SHA384``.
+   :field tls_cipher_bits: Cipher bits, e.g. ``256``.
+   :field tls_pfs: Perfect forward-security mechanism, e.g. ``KxANY``,
+     ``KxECDHE``.
+   :field tls_protocol: TLS protocol name, e.g. ``TLSv1.3``.
+
+.. dovecot_event:field_group:: auth_client_lookup
+   :inherit: auth_client_common
+
+   :field user: Full username to lookup.
+   :field error: Error string, if error occurred.
+
+.. dovecot_event:field_group:: auth_client_request
+   :inherit: auth_client_common
+
+   :field user: Username, if present
+   :field original_user: Original username, if present.
+   :field auth_user: Auth username, if present.
+   :field error: Error string, if error occurred.
+
+
+.. dovecot_core:event:: auth_client_passdb_lookup_started
+   :added: v2.3.7
+   :inherit: auth_client_lookup
+
+   TODO
+
+
+.. dovecot_core:event:: auth_client_passdb_lookup_finished
+   :added: v2.3.7
+   :inherit: auth_client_lookup
+
+   TODO
+
+
+.. dovecot_core:event:: auth_client_request_started
+   :added: v2.3.7
+   :inherit: auth_client_request
+
+   TODO
+
+
+.. dovecot_core:event:: auth_client_request_challenged
+   :added: v2.3.7
+   :inherit: auth_client_request
+
+   TODO
+
+
+.. dovecot_core:event:: auth_client_request_continued
+   :added: v2.3.7
+   :inherit: auth_client_request
+
+   TODO
+
+
+.. dovecot_core:event:: auth_client_request_finished
+   :added: v2.3.7
+   :inherit: auth_client_request
+
+   TODO
```

Comment:
Auth client receives response from server that authentication is finished, either success or failure.